roles/vendors/dell/tasks/iso.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Discovery iDRAC versions for Dell hardware
   containers.podman.podman_container:
-    name: "{{ bmc_address }}-rac-version"
+    name: "{{ bmc_name }}-rac-version"
```

Comment:
Sorry, one thing to change here, please add a no_log: true to this task. Thanks!